code/modules/mob/living/simple_animal/friendly/pet.dm

```diff
@@ -5,6 +5,7 @@
 	var/collar_type
 	var/unique_pet = FALSE
 	blood_volume = BLOOD_VOLUME_NORMAL
+	throwforce = 0
```

Comment:
0 throwforce doesn't make too much sense to me here

Reply:
Caks are a subtype of pets - and there's no reason it shouldn't apply to Ian or other pets, IMO. I can just move it to caks, if you like, but it seems a little silly.

Reply:
It doesn't make too much sense that caks would be 0 throwforce either

Reply:
The main point of this was fixing that the bar shuttle could throw a mob into something or someone about 80 times a second if that mob could climb tables, resulting in the almost-instant fuelsplosion. An alternative would just be making the bar table teleport, rather than throw, backward - it'd mean less touching throw code.

Reply:
Having a way to change the throw force of a mob is good, but you really should just change how the bar tables handle keeping people off them instead of adding this weird work-around that only makes sense in this one scenario.

Reply:
Will do.